pyatlan/cache/user_cache.py

```diff
@@ -92,8 +93,20 @@ def _get_id_for_name(self, name: str) -> Optional[str]:
         :param name: human-readable name of the user
         :returns: unique identifier (GUID) of the user
         """
-        if user_id := self.map_name_to_id.get(name):
+        user_id = self.map_name_to_id.get(name)
```

Comment:
Lines can be consolidated to:
`if user_id := self.map_name_to_id.get(name):`
`    return user_id`